src/scroll-timeline-base.js

```diff
@@ -780,16 +777,36 @@ function calculateInset(value, sizes) {
   return { start, end };
 }
 
+// Calculate the fractional offset of a range value relative to the normal range.
+export function fractionalOffset(timeline, value) {
```

Comment:
FYI: I renamed this function from `relativePosition` to `fractionalOffset` for two reasons:

1. More clearly expresses what it does
2. There’s already a `relativePosition` function in another file present.